### DISCUSSION TITLE
SpinalHDL online meetup

### DISCUSSION THREAD

**Discussion by Dolu1990** [SpinalHDL/SpinalHDL#753] [Announcements]
Hi,

I'm scheduling an online meeting to talk about two things : 
- An eventual SpinalHDL https://opencollective.com/ in order to collect donation 
- Meet each others ^^

Question is who would be interested, and to know your time-zone / availability for the meeting in order to schedule it.

I added a doodle to setup a date : 
https://doodle.com/meeting/participate/id/eERADB0a/vote

Regards,
Dolu1990

See https://groups.google.com/g/spinalhdl-hardware-description-language/c/sZb8gm8PHyM

**Comment by saahm** [comment]
In https://groups.google.com/g/spinalhdl-hardware-description-language/c/sZb8gm8PHyM you write
```
Picked a date :)   (https://doodle.com/meeting/organize/id/eERADB0a)

The meetup will be on https://meet.jit.si/SpinalHDL_meetup

Cya soon ^^
```

I cant see which date got approved and you dont mention it. Can you update us on when its gonna happen? :)

**Reply by Dolu1990** [reply]
Hi Saahm,

It is in 1h and 20 mn (10 am UTC)
Hope you will see that message :)

You can't access https://doodle.com/meeting/organize/id/eERADB0a ?
There is the date on it (at least to me, even on inconito browser mode)

**Reply by saahm** [reply]
It didnt show me anything there other than "This meeting is not available anymore or doesnt exist" in normal browser mode. 

![Without incognito](https://user-images.githubusercontent.com/50137875/172573698-c6af9374-c448-4c4f-babe-4f25181f0502.png)

Since you mentioned it I have looked at it with incognito and it works suddenly... probably because it has cached/cookied that I have added my appointment there???

![With incognito](https://user-images.githubusercontent.com/50137875/172573773-ea690314-7b58-4d98-82f7-85a13596cddb.png)

Note to self: Always check with incognito aswell. Thanks!
